index.html

```diff
@@ -66,11 +64,242 @@
+              </div>
+              <div class="col-3">
+                6
+                <div class="gradient-box gradient-box--6 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                7
+                <div class="gradient-box gradient-box--7 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                8
+                <div class="gradient-box gradient-box--8 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                9
+                <div class="gradient-box gradient-box--9 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                10
+                <div class="gradient-box gradient-box--10 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                11
+                <div class="gradient-box gradient-box--11 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                12
+                <div class="gradient-box gradient-box--12 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                13
+                <div class="gradient-box gradient-box--13 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                14
+                <div class="gradient-box gradient-box--14 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                15
+                <div class="gradient-box gradient-box--15 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                16
+                <div class="gradient-box gradient-box--16 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                17
+                <div class="gradient-box gradient-box--17 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                18
+                <div class="gradient-box gradient-box--18 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                19
+                <div class="gradient-box gradient-box--19 ratio-box--16-9"></div>
+              </div>
+              <div class="col-3">
+                20
+                <div class="gradient-box gradient-box--20 ratio-box--16-9"></div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="section" id="Filters">
+        <div class="container">
+          Fileters
+          <div class="row">
+            <div class="col-4">
+              1
+              <div class="filter-box filter-box--1 ratio-box--16-9">
+                <img src="images/filter1.jpg" alt="">
+              </div>
+            </div>
+            <div class="col-4">
+              2
+              <div class="filter-box filter-box--2 ratio-box--16-9">
+                <img src="images/filter2.jpg" alt="">
```

Comment:
you don't heed more than 2 images, all filter boxes should have the same image to them so please remove all extra images from your folder.
Instead you should have 1 image but in 2 variants: a bigger and a smaller variant of the same image and use `srcset` to apply 1x or 2x